lib/ts/recipe/session/utils.ts

```diff
@@ -239,6 +267,7 @@ export function validateAndNormaliseUserInput(
         errorHandlers,
         antiCsrf,
         override,
+        invalidClaimStatusCode: config?.invalidClaimStatusCode ?? 403,
```

Comment:
need to check that this is not the same as the session expired status code - if it is, throw an error